scripts/install_dev_tools.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+#
+# Copyright 2022 Red Hat, Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+# Match the .venv created by the Makefile
+DEFAULT_VENV="${SCRIPT_DIR}/../.venv"
+
+TMP_DIR_PREFIX="pelorus_tmp_"
+
+TKN_CLIENT_API_URL="https://api.github.com/repos/tektoncd/cli/releases/latest"
+CT_CLIENT_API_URL="https://api.github.com/repos/helm/chart-testing/releases/latest"
+
+HELM_INSTALL_SCRIPT="https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3"
+
+# Currently we are interested in development on Linux and Mac OS platforms
+# Because OCP_CLIENT URL links do not follow standard binary name format
+# e.g. Linux/Darwin we need to map those 
+PLATFORM="linux"
+[[ "$(uname -s)" =~ "Darwin" ]] && PLATFORM="mac"
+OCP_CLIENT_URL="https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-${PLATFORM}.tar.gz"
+
+
+function download_file_from_url() {
+    local url=$1
+    local dest_folder=$2
+
+    pushd "${dest_folder}" || exit
+      echo "Downloading file: ${url}"
+      echo "To: ${dest_folder}"
+      curl -LO "${url}"
+    popd || exit
```

Comment:
These exits should be an error, right?

Reply:
Already is:
```
$ TEST=$(pushd /tmp/nonexiting || exit)
$ echo $TEST
1
```

Reply:
Ah right, I forgot it uses the last exit code.